qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -167,10 +167,9 @@ def _checks(self, run_seq):
         log.info("Cloning repo into place")
         repo_path = TestScrubChecks.clone_repo(self.mount_a, client_path)
 
-        log.info("Initiating mds_scrub_checks on mds.{id_}, " +
-                 "test_path {path}, run_seq {seq}".format(
-                     id_=mds_rank, path=abs_test_path, seq=run_seq)
-                 )
+        log.info("Initiating mds_scrub_checks on mds.{id_} test_path {path}, run_seq {seq}".format(
```

Comment:
i think the reason why we wrapped the line is to please requirement like https://www.python.org/dev/peps/pep-0008/#id19 .

Reply:
But we dont' force it (eg. with flake8) so I don't see a problem here. Or do you want me to change it back?

Reply:
i don't feel strong on this. just wanted to make sure we are improving the quality of code or or making it compliant to a sound policy/guideline.